examples/test/BUILD.bazel

```diff
@@ -0,0 +1,9 @@
+cc_test(
```

Comment:
Nit: Can you think of a better name for the folder than test, should it be testing?

Reply:
I think the name should describe the content of the folder, this folder contains a test file so it is called 'test'. I think this sufficiently captures the required information.

I don't see what we gain by renaming it to 'testing', it just converts a sufficiently descriptive noun to a verb in present continuous tens, but for what purpose?